runc.go

```diff
@@ -765,14 +766,23 @@ func (r *Runc) Features(context context.Context) (*features.Features, error) {
 	return &feat, nil
 }
 
-func (r *Runc) args() (out []string) {
+func (r *Runc) args(logPath string) (out []string) {
 	if r.Root != "" {
 		out = append(out, "--root", r.Root)
 	}
 	if r.Debug {
 		out = append(out, "--debug")
 	}
+
 	if r.Log != "" {
+		if logPath != "" {
+			out = append(out, "--log", logPath)
+		} else {
+			out = append(out, "--log", r.Log)
+		}
+	}
+
+	if r.Log == "" && logPath != "" {
 		out = append(out, "--log", r.Log)
```

Comment:
Was this intended to append `logPath`, not `r.Log` (empty) ?

Reply:
nod missed it.. will ask for a unit bucket